src/Microsoft.AspNetCore.Sockets.Http/ConnectionManager.cs

```diff
@@ -53,7 +53,14 @@ public void Start()
 
         public bool TryGetConnection(string id, out DefaultConnectionContext connection)
         {
-            return _connections.TryGetValue(id, out connection);
+            connection = null;
+
+            if (_connections.TryGetValue(id, out var pair))
```

Comment:
Would `return _connections.TryGetValue(id, out (connection, _));` work? Destructure all the things?!

Reply:
Ugh,